jobs/build/update-3.x-base-images/build.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+#➜  ~ docker run -it brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/jboss/openjdk18-rhel7:latest id
+#uid=185(jboss) gid=185(jboss) groups=185(jboss)
+#➜  ~ docker run -it  brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/elasticsearch:latest id
+#uid=1000(elasticsearch) gid=1000(elasticsearch) groups=1000(elasticsearch)
+#➜  ~  docker run -it brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/ansible-runner:latest  id
+#uid=0(root) gid=0(root) groups=0(root)
+# `rhel7` is root 0`
+# `nodejs6` is 1001`
+
+build_common() {
+    TAG="rhaos-4.0-rhel-7"
+    URL="http://pkgs.devel.redhat.com/cgit/containers/openshift-enterprise-base/plain/.oit/signed.repo?h="
+    USER_USERNAME="--user=ocp-build"
+    REPO=${URL}+${TAG}
+    TARGET_DIR=build-$1
+    rm -rf ${TARGET_DIR}
+    rhpkg ${USER_USERNAME} clone --branch ${TAG} containers/openshift-enterprise-base ${TARGET_DIR}
+    cd ${TARGET_DIR}
+    echo "$1" > additional-tags
+    echo """FROM $2
+
+    USER root
+    RUN yum update -y && yum clean all
+    USER $3
+
+    LABEL \\
+            com.redhat.component=\"openshift-enterprise-base-container\" \\
+            name=\"openshift/ose-base\" \\
+            version=\"v4.0\" \\
+            release=\"$(date +%Y%m%d%H%M)\"
+    """ > Dockerfile
+    git commit -am "updated $1 container"
+    git push
+    rhpkg  ${USER_USERNAME} container-build --repo-url ${REPO}
+}
+
+#build_common package_name image_name uid
+build_common ansible-runner ansible-runner:latest 0
+build_common elasticsearch elasticsearch:latest 1000
+build_common jboss.openjdk18.rhel7 jboss/openjdk18-rhel7:latest 185
+build_common rhscl.nodejs.6.rhel7 rhscl/nodejs-6-rhel7:latest 1001
+build_common rhel7 rhel7:latest 0
```

Comment:
`rhel7` should _not_ be `:latest`. use `:7-released`. same for `ubi7`.
the others are currently `:latest` only because they don't update frequently. For this we should base it on a tag that points to what is currently latest and only update it if we learn there is a reason to.